Makefile

```diff
@@ -0,0 +1,27 @@
+# This rule runs the resource scaffolding script
+# $1 is the name of the resource to generate
+
+generate:
+	@cd ./scripts/generate_resource; \
+	go mod download; \
+	go mod tidy; \
+	go run . $(resource) ;
+
+vet:
+	go vet
+
+lint:
+	golangci-lint run .
+
+test:
+	go test ./... $(TESTARGS) -tags=integration -timeout=30m
+
+mocks:
+	@echo "mockgen -source=$(filename) -destination=mocks/$(filename) -package=mocks" >> generate_mocks.sh
```

Comment:
If I type "make mocks" it will add an empty line to this file: `mockgen -source= -destination=mocks/ -package=mocks`

Is it possible to test these make parameters to make sure they exist?